packages/magic_sdk/pubspec.yaml

```diff
@@ -12,9 +12,9 @@ dependencies:
   flutter:
     sdk: flutter
   package_info_plus: ^3.0.2
-  webview_flutter: ^3.0.4
+  webview_flutter: ^4.2.0
   json_annotation: ^4.5.0
-  web3dart: 2.5.1
+  web3dart: ^2.6.1
```

Comment:
Could you lock the web3dart version as the library is poorly managed, sometimes they introduced breaking changes with minor version bumps?

Reply:
One caveat is, when you lock the version, the flutter linter will say your implementation is bad because you lock the version. I think we'll just have to bite the bullet